scripts/patch.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cp .patches/options.patch.go vendor/github.com/google/go-containerregistry/pkg/crane/.
+cp .patches/getter.patch.go vendor/helm.sh/helm/v3/pkg/getter/.
+cp .patches/action.patch.go vendor/helm.sh/helm/v3/pkg/action/.
+cp .patches/install.patch.go vendor/helm.sh/helm/v3/pkg/action/.
+patch -p1 -N -i .patches/helm.patch
```

Comment:
We could write the `.rej` file to `/dev/null` to avoid polluting `vendor` even further.
```suggestion
patch -p1 -r /dev/null -N -i .patches/helm.patch
```

Reply:
Fixed.